### PR TITLE
Update FastJX and SuperFast

### PR DESCRIPTION
**Pull request by ctessum**

### CODE REVIEW

File: Project.toml [hunk 1]
```diff
@@ -7,15 +7,19 @@ version = "0.2.0"
 BenchmarkTools = "6e4b80f9-dd63-53aa-95a3-0cdb28fa8baf"
 Catalyst = "479239e8-5488-4da2-87a7-35f2df7eef83"
 Dates = "ade2ca70-3891-5945-98fb-dc099432e06a"
+
+DifferentialEquations = "0c46a032-eb83-5123-abaf-570d42b7fbaa"
```
Comment (ctessum) [inline Project.toml:11]: These three packages are already listed below under "extras", and then under "targets" it specifies that these packages should only be available at test time. So we can delete them from up here, which will make things run slightly faster.

File: src/Fast-JX.jl [hunk 2]
```diff
@@ -1,9 +1,7 @@
-export FastJX
+export FastJX, j_mean_NO2, j_mean_o31D, j_mean_H2O2, j_mean_CH2Oa, j_mean_CH3OOH
```
Comment (ctessum) [inline src/Fast-JX.jl:1]: Is there a reason that users would need easy access to the `j_mean` functions? If not, then we shouldn't export them.

File: src/Fast-JX.jl [hunk 3]
```diff
@@ -150,13 +165,13 @@ struct FastJX <: EarthSciMLODESystem
...
+            j_CH2Oa ~ j_mean_CH2Oa(t, lat, long, T) * j_unit
+            j_o31D ~ j_mean_o31D(t, lat, long, T) * j_unit
+            j_CH3OOH ~ j_mean_CH3OOH(t, lat, long, T) * j_unit
+            j_NO2 ~ j_mean_NO2(t, lat, long, T) * j_unit
         ]
 
-        new(ODESystem(eqs, t, [j_h2o2, j_CH2Oa, j_o31D, j_CH3OOH, j_NO2], [lat, j_unit, T]; name=:fastjx))
+        new(ODESystem(eqs, t, [j_h2o2, j_CH2Oa, j_o31D, j_CH3OOH, j_NO2], [lat, long, j_unit, T]; name=:fastjx))
     end
 end
```
Comment (ctessum) [inline src/Fast-JX.jl:177]: Do we also have code for coupling SuperFast and FastJX together? If so, we can add that here as well.

**Review by ctessum — COMMENTED** [review]
Looking good! Just a couple of questions/comments.

File: src/Fast-JX.jl (hunk 3 above)
Comment (jialinl6) [reply src/Fast-JX.jl:177]: Do we want to delete the file "src/compose_fastjx_superfast," which contains the code for coupling SuperFast and FastJX, and instead move that code here?

Comment (ctessum) [reply src/Fast-JX.jl:177]: Oh, nevermind, I forgot about that.

**Comment by ctessum** [comment]
Looks good, thanks!

**Pull request merged**